ios/FluentUI/Extensions/EnvironmentValues+Extensions.swift

```diff
@@ -23,3 +23,20 @@ private extension UIEdgeInsets {
         EdgeInsets(top: top, leading: left, bottom: bottom, trailing: right)
     }
 }
+
+struct ViewControllerHolder {
+    weak var value: UIViewController?
+}
+
+struct ViewControllerKey: EnvironmentKey {
+    static var defaultValue: ViewControllerHolder {
+        return ViewControllerHolder(value: UIApplication.shared.windows.first?.rootViewController)
```

Comment:
`-[UIApplication windows]` is deprecated; we'll want to use a different API to get a default window (maybe back to the key window?)